packages/block-editor/src/components/block-draggable/index.js

```diff
@@ -76,6 +76,7 @@ const BlockDraggable = ( {
 		<Draggable
 			cloneClassname={ cloneClassname }
 			elementId={ elementId }
+			__experimentalTransferDataType="wp-blocks"
```

Comment:
Would you like me to remove the type key inside the data itself? I was never sure why it was there.

Reply:
if it's possible to remove yes, we already use at least two types there.

Reply:
Two types? 🤔 I'll check when I remove it.

Reply:
Hm, ok, I see those two subtypes "inserter" and "block". I'd prefer to look at that in a separate PR because it doesn't really seem relevant here. The reason I'm switching the `dataTransfer` [format](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setData#arguments) to `wp-blocks` is to make it easier to differentiate it from normal text, html and files dragging. Currently we set it as normal text (which isn't right in the first place), so we'd need to parse the text data simply to know what it is.

Reply:
> The reason I'm switching the dataTransfer format to wp-blocks is to make it easier to differentiate it from normal text, html and files dragging. Currently we set it as normal text (which isn't right in the first place), so we'd need to parse the text data simply to know what it is.

This makes me wonder if this type should be more something like `wp` (not blocks but just something to indicate that it's the components drag/drop) and avoid the prop entirely.

Reply:
Not sure I'm following. What should be `wp`? The `dataTransfer` format?

Reply:
Nevermind, it's not that important. I still think we need to dedupe the types but I'm ok with it being a follow-up.

Reply:
Ok, sounds good! 🙂